core/src/main/java/org/geysermc/geyser/GeyserImpl.java

```diff
@@ -681,6 +681,12 @@ public BedrockListener bedrockListener() {
         return getConfig().getBedrock();
     }
 
+    @Override
+    @NonNull
```

Comment:
Override below nullability?

Reply:
The rest of them in the class are this way round so I think it should be left

Reply:
o i didn't notice, agreed